UPGRADING.md

```diff
@@ -0,0 +1,164 @@
+# Upgrading Kwil
+
+This guide provides instructions for upgrading to specific [versions](https://github.com/kwilteam/kwil-db/releases) of `kwild`, `kwil-cli`, and `kwil-admin`. It also includes information on on changes to the Core Module (Go SDK).
+
+## v0.8.0
+
+Kwil v0.8 introduces substantial changes to the Kuneiform language (SQL smart contracts), an improved network migration process, a system for coordinating consensus rules changes, and a JSON-RPC server.
+
+### Upgrading from v0.7.x
+
+Nodes running `kwild` v0.8 are not compatible with nodes running `kwild` v0.7.
+
+If a v0.7 network uses schemas that **are** compatible with v0.8, follow the [network migrations guide](https://docs.kwil.com/docs/node/network-migrations) to export the network data (CometBFT and Postgres) and import it into a new v0.8 network.
```

Comment:
It does not export comet data

Reply:
Also, maybe add a note here to see the breaking changes to determine whether a schema is compatible?

Reply:
Also, to test whether a schema is compatible, users should, using the `kwil-cli` for v0.8, run `kwil-cli utils parse ./schema.kf`. If it parses with no errors, their schema is compatible.